lms/templates/footer.html

```diff
@@ -103,10 +103,10 @@
 
         <div class="wrapper-logo">
             <div class="navbar-brand">
-                <a href="https://web.mit.edu/" rel="noopener noreferrer" target="_blank" class="mit-link"></a>
-                <a href="${settings.MARKETING_SITE_BASE_URL}"" class="marketing-site-link"></a>
-                <img  class="logo xpro-logo" src="${footer['logo_image']}" alt="${_("{platform_name} Home Page").format(platform_name=static.get_platform_name())}"/>
-            </div>
+                <a href="${settings.MARKETING_SITE_BASE_URL}"" class="marketing-site-link">
```

Comment:
@cachob The footer logo currently has two links. Check the footer at https://courses-rc.xpro.mit.edu/courses/course-v1:MITx+AMxB2+R5/instructor#view-course_info.

We are now replacing it with the MIT Open Learning logo. (See screenshot). Where do you think this logo should redirect? Currently, It will redirect the user to the xPRO application e.g. https://rc.xpro.mit.edu/.